### ISSUE TITLE
License chooser doesn't differentiate between GPLv{2,3}{,+}

### ISSUE THREAD

**Issue by lfaraone** [todogroup/gh-issues#24]
When specifying a GNU license, just a LICENSE file is committed. This means that all new projects become GPLv2-only or GPLv3-only. 

It'd be nice to offer options for GPLv2+ vs GPLv2, and make the forward-compatible option more prominently recommended.

**Comment by jeffmcaffer** [comment]
Worthwhile seeing what's going on here. Is this a limitation of Licensee (i.e., working backwards from the text to know V2.0+) or is it an effort to keep license selection simple. 

/cc @benbalter may be interested or have views

**Comment by byjrack** [comment]
Yeah while the author may know where the "or later" is declared it may be complicated outside of assumptions in the COPYING.md
